cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -111,8 +111,7 @@ def _serialize_circuit(
             if self.use_constants_table_for_moments:
                 if moment_index := raw_constants.get(moment, None):
```

Comment:
Let us handle corner case when `moment` is at index `0`.
```suggestion
                if moment_index := raw_constants.get(moment, None) is not None:
```
same for the similar constructs at lines 112, 135.

Reply:
Good call.  Done.